viewing_party/party.py

```diff
@@ -1,23 +1,231 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+
+    #return None of title, genre, and rating are empty
+    if not (title and genre and rating):
+        return None
+    
+    movie = {
+                "title": title,
+                "genre": genre,
+                "rating": rating
+            
+            }
+    return movie
+
+
+def add_to_watched(user_data, movie):
+
+    user_data["watched"].append(movie)
+    
+    return user_data

```

Comment:
Finding a balance of whitespace inside of a function can be hard, and is often subjective. That said, for a very short function like this with a single statement and return, I would recommend removing the new lines on line 19 & 21 since this function is so small that everything is tightly related. This applies to the add_to_watchlist function below as well.